xonsh/base_shell.py

```diff
@@ -15,11 +13,10 @@
 
 
 class TeeOut(object):
-    """Tees stdout into the original sys.stdout and another buffer instance that is 
-    provided.
-    """
+    """Tees stdout into the original sys.stdout and another buffer."""
 
     def __init__(self, buf, *args, **kwargs):
+        _, _ = args, kwargs
```

Comment:
Upon further inspection I see that the TeeOut and TeeErr classes are only instantiated from class Tee and are only called with one positional argument. So why do they need the *args and **kwargs arguments? Their interfaces are only relevant to another class in the same module and thus do not need "future proofing". Too, shouldn't they be private to the module? Only the Tee class is meant to implement the io.StringIO interface.

This type of issue is why I think the code should be lint clean. Most of the errors pylint reports are legitimate. The few that are not can be suppressed via pylint directives.